public/app/percona/dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -53,20 +60,50 @@ export const KubernetesInventory: FC<KubernetesInventoryProps> = ({ setMode }) =
   const { result: kubernetes, loading: kubernetesLoading } = useSelector(getKubernetesSelector);
   const { loading: deleteKubernetesLoading } = useSelector(getDeleteKubernetes);
   const { loading: addKubernetesLoading } = useSelector(getAddKubernetes);
-  const loading = kubernetesLoading || deleteKubernetesLoading || addKubernetesLoading;
+  const [update, setUpdate] = useState(false);
+  const [catchFromAsyncThunkAction] = useCatchCancellationError();
+
+  const loading = useMemo(
+    () => (kubernetesLoading || deleteKubernetesLoading || addKubernetesLoading) && !update,
+    [kubernetesLoading, deleteKubernetesLoading, addKubernetesLoading, update]
+  );
+  const k8sListShouldBeUpdated = useMemo(
```

Comment:
Both these memos depend on state/store vars already. This means we can use simple const and they will change when those vars change, because they cause the re-render of the component, so `useMemo` shouldn't be necessary.

Reply:
fixed